plugins/slack/src/index.ts

```diff
@@ -39,7 +41,8 @@ export default class SlackPlugin implements IPlugin {
     } else {
       this.options = {
         url: options.url ? options.url : '',
-        atTarget: options.atTarget ? options.atTarget : 'channel'
+        atTarget: options.atTarget ? options.atTarget : 'channel',
+        publishPreRelease: options.publishPreRelease ? options.publishPreRelease : false
```

Comment:
I dont think the ternary adds anything. I think it could just be the following 

```suggestion
        publishPreRelease: options.publishPreRelease
```